core/network/fabric-interop-cc/contracts/interop/write_external_state_test.go

```diff
@@ -19,10 +19,77 @@ import (
 	wtest "github.com/hyperledger-labs/weaver-dlt-interoperability/core/network/fabric-interop-cc/libs/testutils"
 )
 
+<<<<<<< HEAD
+=======
```

Comment:
Hey I think the merge messed up the source code at other places, can you undo it?